backend/app/services/memory/client.py

```
@@ -0,0 +1,351 @@
+            True if service is reachable and healthy, False otherwise
+        """
+        try:
+            session = await self._get_session()
+
+            async with session.get(
+                f"{self.base_url}/health",
+                timeout=aiohttp.ClientTimeout(total=5.0),
+            ) as resp:
+                return resp.status == 200
+
+        except Exception as e:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)